src/coreComponents/schema/docs/Equilibrium.rst

```diff
@@ -0,0 +1,26 @@
+
+
+============================================ ============ ======== ==================================================================================================== 
+Name                                         Type         Default  Description                                                                                          
+============================================ ============ ======== ==================================================================================================== 
+bcApplicationTableName                       string                Name of table that specifies the on/off application of the bc.                                       
```

Comment:
```suggestion
bcApplicationTableName                       string                Name of table that specifies the on/off application of the boundary condition.                                       
```

Reply:
@sytuannguyen this table is automatically generated.